### PR TITLE
Ft nyurwemart.rw

### PR DESCRIPTION
**Pull request by pextech**
what does this PR do?
   - it contains the nyurwemart.rw websitr

how to manually access this feature?
   - clone this repo https://github.com/KitechRW/akadomorw.git
   - navigate to branch ft-nyurwemart.rw
   - run the file in the browser
what are the relevant trello stories
   - https://trello.com/c/bfdGtXvm

### CODE REVIEW

File: template/script.js [hunk 1]
```diff
@@ -38,40 +38,36 @@ function includeHTML() {
...
-	).innerHTML = `Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero magni asperiores velit!`;
+	).innerHTML = ` We are working constantly to go above and beyond customer expectations!`;
 
 	// right section in about us
 	document.querySelector(
 		'#visions'
-	).innerHTML = `Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero magni asperiores velit!`;
+	).innerHTML = `Our Vision. A vibrant and healthy community with access to healthy food through an affordable, member-owned, grocery store`;
 
 	// In team section
```
Comment (fridolinho) [inline template/script.js:72]: remove team section scripts they prevent other scripts to run successfully since they are not needed.

**Review by fridolinho — COMMENTED** [review]
footer text are not seen since the background and text are all black

File: template/script.js (hunk 1 above)
Comment (pextech) [reply template/script.js]: done

File: template/index.html [hunk 2]
```diff
@@ -1,39 +1,36 @@
...
+
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+		integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous" />
+	<link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+	<script src="https://use.fontawesome.com/c28c4f7255.js"></script>
+	<link rel="stylesheet" href="style.css" />
+	<link rel="shortcut icon" href="#" type="image/x-icon">
```
Comment (fridolinho) [inline template/index.html:12]: add nyurwemart logo location as href value to set the favicon

File: template/components/footer.html [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <div class="common-bg-color text-center p-4">
-	<span class="text-white">Kitech &copy; 2021</span>
-</div>
+	<span class="text-white">Nyurwe mart &copy; 2021</span>
```
Comment (fridolinho) [inline template/components/footer.html:2]: Text are not readable

**Review by fridolinho — COMMENTED** [review]
Kindly make the 2 changes
1. footer text
2. favicon

File: template/components/contact.html [hunk 4]
```diff
@@ -79,50 +59,20 @@
...
-					<span class="badge common-bg-color icons-container"
-						><a
-							id="linkedin"
-							target="_blank"
-							class="fa fa-linkedin font-awesome-underline social-icons"
-						></a
-					></span>
+					<span class="badge common-bg-color icons-container"><a id="facebook" target="_blank"
+							class="fa fa-facebook font-awesome-underline social-icons"></a></span>
+					<span class="badge common-bg-color icons-container"><a id="twitter" target="_blank"
```
Comment (fridolinho) [inline template/components/contact.html:64]: remove twitter link and the reference in the script

File: template/components/contact.html [hunk 5]
```diff
@@ -79,50 +59,20 @@
...
+							class="fa fa-twitter font-awesome-underline social-icons"></a></span>
+					<span class="badge common-bg-color icons-container"><a id="instagram" target="_blank"
+							class="fa fa-instagram font-awesome-underline social-icons"></a></span>
+					<span class="badge common-bg-color icons-container"><a id="whatsapp" target="_blank"
+							class="fa fa-whatsapp font-awesome-underline social-icons"></a></span>
+					<span class="badge common-bg-color icons-container"><a id="youtube" target="_blank"
```
Comment (fridolinho) [inline template/components/contact.html:70]: the same for youtube

File: template/components/contact.html [hunk 6]
```diff
@@ -79,50 +59,20 @@
...
+							class="fa fa-youtube font-awesome-underline social-icons"></a></span>
+					<span class="badge common-bg-color icons-container"><a id="linkedin" target="_blank"
```
Comment (fridolinho) [inline template/components/contact.html:72]: the same for linkedin

File: template/script.js [hunk 7]
```diff
@@ -38,69 +38,48 @@ function includeHTML() {
...
 
 	// Link to social media
-	document.querySelector('#facebook').href = 'http://www.facebook.com';
+	document.querySelector('#facebook').href = 'https://www.facebook.com/105796641400126/posts/121394863173637/?substory_index=0&app=fbl';
 	document.querySelector('#twitter').href = 'http://www.twitter.com';
-	document.querySelector('#instagram').href = 'http://www.instagram.com';
-	document.querySelector('#whatsapp').href = 'https://wa.me/+250788384757';
+	document.querySelector('#instagram').href = 'https://www.instagram.com/p/CKZaXr9Hi8v/?igshid=18563hzvho3xr';
+	document.querySelector('#whatsapp').href = 'https://wa.me/+250782877770';
 	document.querySelector('#youtube').href = 'http://www.youtube.com';
```
Comment (fridolinho) [inline template/script.js:83]: remove twitter script, the same for youtube and linkedin

**Review by fridolinho — COMMENTED** [review]
more feedback on social icons remove unused social media twitter, youtube and linkedin

**Pull request closed without merging**